Jenkinsfile

```diff
@@ -31,7 +51,7 @@ node {
 
       govuk.setEnvar("RAILS_ENV", "test")
       govuk.bundleApp()
-      
+
       govuk.contentSchemaDependency()
```

Comment:
You should pass `env.SCHEMA_BRANCH` as a parameter to this function so that the correct branch of the schemas is checked out. Otherwise it defaults to `deployed-to-production`, which is correct for branch builds of _this_ project, but incorrect for the schema tests.

I realise I forgot to add this to the service manual instructions, and I also missed it in some of the Jenkinsfiles that I created! 😱  I'll go back and update them, but in the meantime here's an example of the correct config: https://github.com/alphagov/collections-publisher/blob/master/Jenkinsfile

Otherwise, this all looks good.